SPECS/fakeroot/fakeroot.spec

```diff
@@ -182,6 +179,9 @@ fi
 %ghost %{_libdir}/libfakeroot/libfakeroot-0.so
 
 %changelog
+* Fri Mar 18 2022 Pawel Winogrodzki <pawelwi@microsoft.com> - 1.28-1
+- Updating to version 1.28 using Fedora 36 spec (license: MIT) for guidance.
+
```

Comment:
Maybe a chagelog entry that we modified `relax_tartest.patch` to be on parity with upstream :)